modules/setting/lfs.go

```diff
@@ -40,18 +40,6 @@ func loadLFSFrom(rootCfg ConfigProvider) error {
 
 	lfsSec, _ := rootCfg.GetSection("lfs")
 
-	// Specifically default PATH to LFS_CONTENT_PATH
-	// DEPRECATED should not be removed because users maybe upgrade from lower version to the latest version
-	// if these are removed, the warning will not be shown
-	deprecatedSetting(rootCfg, "server", "LFS_CONTENT_PATH", "lfs", "PATH", "v1.19.0")
```

Comment:
Can we still keep them at the original places? I think that will help code readers to find the history of the code.